lib/new_relic/agent/llm_event/chat_completion.rb

```diff
@@ -0,0 +1,28 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+module NewRelic
+  module Agent
+    class LlmEvent
+      class ChatCompletion < LlmEvent
```

Comment:
Is `ChatCompletion` meant to be nested inside the `LlmEvent` class AND inherit from it as well?